push/templates/push/list.html

```diff
@@ -71,6 +71,7 @@
                             <th>{% trans "Application Name" %}</th>
                             <th>VAPID {% trans "Key" %}</th>
                             <th>VAPID {% trans "Key Status" %}</th>
+                            <th></th>
```

Comment:
Yeah, I had a label in here, but the most intuitive label seemed to be "Delete" which was redundant with all the "Delete" buttons in the column on every row. I also liked leaving the "Validate" button in the "VAPID Key Status" column since that action is associated with the status.

What do you think - okay to leave the column header blank until we have another button to include in it? Or should we move the "Validate" button to this column and label it "Actions" ?